lib/kernels/clink_kernels.cc

```diff
@@ -64,19 +52,38 @@ AsyncValueRef<double> SquareAdd(Argument<double> x, Argument<double> y,
 
 double Square(double x) { return x * x; }
 
-void OneHotEncoderLoad(Argument<std::string> path,
-                       Result<RCReference<OneHotEncoderModel>> result_model,
-                       KernelErrorHandler handler,
-                       const ExecutionContext &exec_ctx) {
-  auto model = OneHotEncoderModel::load(path.get(), exec_ctx.host());
-  CLINK_RETURN_IF_ERROR(handler, model.takeError());
-  result_model.Emplace(model.get());
+template <class T>
+void ModelLoad(Argument<std::string> path,
+               Result<RCReference<Model>> result_model,
+               KernelErrorHandler handler, const ExecutionContext &exec_ctx) {
+  auto result = result_model.Allocate();
+  bool work_enqueued = tfrt::EnqueueBlockingWork(
```

Comment:
nits: could we add a comment here explaining why we use `EnqueueBlockingWork(...)`?